lib/robot.js

```diff
@@ -25,6 +25,25 @@ class Robot {
     });
   }
 
+  addEventToListeners(event, callback) {
```

Comment:
Does this need to be a class method, or can it be a function? If its a class method, then it'll be accessible by calling `robot.addEventToListeners()` whether we want it to be or not.